python/tank/bootstrap/configuration_writer.py

```diff
@@ -34,7 +34,7 @@ class ConfigurationWriter(object):
     Class used to write and update Toolkit configurations on disk.
     """
 
-    def __init__(self, path, sg):
+    def __init__(self, path, sg, is_unmanaged=True):
```

Comment:
sorry for the delay here. i neeed time to think things through.

unmanaged config is a config which doesn't have a pipeline configuration entity in shotgun. I am worried we are blurring the lines here by using unmanaged in this way.

I have some suggestions which are all subtle but hopefully useful - would love to hear your thoughts about them!